CHANGELOG.md

```diff
@@ -3,3 +3,5 @@
 ### Features
 
 ### Breaking Changes
+
```

Comment:
This needs to be changed, we now need to move this into the 0.18 section (please create it).

Reply:
ok

Reply:
Actually I have a PR here https://github.com/instructlab/instructlab/pull/1427 so I stop asking people about it.

Reply:
The new release should be at the top of the file.